packages/cactus-plugin-ledger-connector-xdai/src/test/typescript/integration/openapi/openapi-validation.test.ts

```diff
@@ -163,15 +163,15 @@ test(testCase, async (t: Test) => {
         gas: 1000000,
       };
       await apiClient.deployContractV1(
-        (parameters as any) as DeployContractV1Request,
+        (parameters as unknown) as DeployContractV1Request,
```

Comment:
@aldousalvarez Thanks for the updates so far. I missed this one earlier, sorry. Same deal here as with the other API call: The cast should not be necessary at all because the correct input object has to be used anyway, so please update this the same way as you did with the other one.